src/open_producten/producttypen/models/producttype.py

```diff
@@ -4,6 +4,7 @@
 from django.db import models
 from django.utils.translation import gettext_lazy as _
 
+from django_json_schema_model.models import JsonSchema
```

Comment:
I'll put here my questions about code for `django_json_schema_model.models.JsonSchema`:
1. It looks like `name` field should be unique
1. Why do you use `Draft202012Validator` in the clean() instead of using `$schema` keyword defined in the JSON schema?

Reply:
Instead of `validate("{}", self.schema)` you mean? 

I felt like Draft202012Validator.check_schema(self.schema) was cleaner and it is also what is called within `jsonschema.validate` (third party package)

Reply:
I meant `jsonschema.validators.validator_for(schema)` - it will return the validator class based on `$schema` keyword, otherwise you can use JSON schema validation rules not applicable for a particular schema